Makefile

```diff
@@ -85,6 +85,9 @@ ifdef KEEP_ISO
     KEEP_ISO_FLAG = --keep-iso
 endif
 
+NUM_SECONDARY_MASTERS := $(or $(NUM_SECONDARY_MASTERS), 0)
+NUM_SECONDARY_WORKERS := $(or $(NUM_SECONDARY_WORKERS), 0)
```

Comment:
same here. Just do it inside code. Why it matters how much workers are on another network?
And the name is really not clear